### DISCUSSION TITLE
4.1.1

### DISCUSSION THREAD

**Discussion by aussig** [aussig/BGS-Tally#273] [Announcements]
### Bug Fixes:

* The Discord report was blank if the Discord language was set to anything other than English
* INF numbers in legend / help window weren't showing the new ones implemented in v4.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/aussig/BGS-Tally/releases/tag/4.1.1'>4.1.1</a>.</em>

**Comment by NilesSeverin** [comment]
I'm not sure if this is the right place to post this, so my apologies if it is inappropriate.

BGS Tally refuses to re-authenticate my API key since the update. Using both Frontier and Steam authenticators, I send the request from the authenticators for BGS Tally to apply the new key, but nothing happens.

Can you advise?

**Comment by aussig** [comment]
Authentication has nothing to do with BGS-Tally - that’s all handled by EDMC. 

The EDMC project released v5.12.1 last night, which included a potential bug fix for this issue. Try updating to that version.
